app/views/team/admin.scala

```diff
@@ -178,13 +178,7 @@ $('#form3-message').val($('#form3-message').val() + e.target.dataset.copyurl + '
               case (remaining, until) =>
                 frag(
                   p(cls := (remaining <= 0).option("error"))(
-                    "You can send up to ",
-                    lila.app.mashup.TeamInfo.pmAllCredits,
-                    " team messages per week. ",
-                    strong(remaining),
-                    " messages remaining until ",
-                    momentFromNowOnce(until),
-                    "."
+                    sevenTeamMessagesPerWeek(strong(remaining), momentFromNowOnce(until))
```

Comment:
```suggestion
                    teamMessagesPerWeekLimit(strong(remaining), momentFromNowOnce(until))
```

Can we remove the `seven` from the key too? Just in case the limit changes in the future.